go/pkg/tool/tool_test.go

```diff
@@ -0,0 +1,43 @@
+package tool
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/command"
+	"github.com/bazelbuild/remote-apis-sdks/go/pkg/fakes"
+)
+
+func TestTool_DownloadActionResult(t *testing.T) {
+	e, cleanup := fakes.NewTestEnv(t)
+	defer cleanup()
+	cmd := &command.Command{
+		Args:        []string{"tool"},
+		ExecRoot:    e.ExecRoot,
+		InputSpec:   &command.InputSpec{},
+		OutputFiles: []string{"a/b/out"},
+	}
+	opt := command.DefaultExecutionOptions()
+	want := "output"
+	_, acDg := e.Set(cmd, opt, &command.Result{Status: command.CacheHitResultStatus}, &fakes.OutputFile{Path: "a/b/out", Contents: want},
+		fakes.StdOut("stdout"), fakes.StdErrRaw("stderr"))
```

Comment:
should the tool also download the StdOut/StdErr of the command and print them?

Reply:
Done.